src/codec2_ofdm.h

```diff
@@ -49,7 +49,11 @@ extern "C" {
 #define UN_SYNC      0  /* Used with the ofdm_set_sync() */
 #define AUTO_SYNC    1
 #define MANUAL_SYNC  2
-    
+
+#define AUTO_PHASE_EST 0
+#define LOW_PHASE_EST  1
+#define HIGH_PHASE_EST 2
+
```

Comment:
So why do we need #defines and the enum to label the same variable?

Reply:
> Do we need #defines and the enum to label the same variable? We could perhaps expose the enum or uses the #defines internally.

Currently the enum is in the ofdm_internal and not exposed public. The system calls also expect an integer. Thus, the simplest form is to use a define for public call, and translate to an enum internally.

I'm not hard over on that requirement.